sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md

```diff
@@ -57,7 +57,8 @@ import static com.azure.core.util.tracing.Tracer.PARENT_SPAN_KEY;
 /**
  * Sample demonstrates tracing how to add and list secrets in a Key Vault with tracing enabled with a Logging Exporter.
  */
-public class Sample { 
+public class Sample {
```

Comment:
Should these code samples be in Java files instead and the markdown file just has links to them?